app/models/ems_azure.rb

```diff
@@ -29,17 +30,16 @@ def connect(options = {})
     self.class.raw_connect(clientid, clientkey, tenant_id)
   end
 
-  def verify_credentials(_auth_type = nil, _options = {})
-    # TODO
-    true
-  end
-
-  def tenant_id=(tenant_id)
-    self.uid_ems = tenant_id
-    save
-  end
+  def verify_credentials(*)
```

Comment:
This should have the same method signature as the other `verify_credentials` methods:

* [vmware](https://github.com/ManageIQ/manageiq/blob/a2fb9f02d1ff9e59dad075b0e09bd4cd2c83ad35/app/models/manageiq/providers/vmware/infra_manager.rb#L115)
* [openstack](https://github.com/ManageIQ/manageiq/blob/a2fb9f02d1ff9e59dad075b0e09bd4cd2c83ad35/app/models/manageiq/providers/openstack/manager_mixin.rb#L120)